.github/workflows/nix-linux.yml

```diff
@@ -26,4 +26,3 @@ jobs:
         skipNixBuild: false
         name: hasktorch
         signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
-        nixBuildArgs: -A hasktorch-codegen -A libtorch-ffi_cpu  -A hasktorch_cpu -A hasktorch-examples_cpu -A hasktorch-experimental_cpu
```

Comment:
I'm not sure if this is on purpose, but one problem with this is that none of the tests will be run by nix, because the `default.nix` file doesn't expose any of the `checks`.

Reply:
oh, I've got to do something about that, thanks!

Reply:
This is for testing only CPU's library not CUDA's one to reduce compilation-time, because building everything costs time and disk.

Reply:
CUDA's one is here. https://github.com/hasktorch/hasktorch/blob/master/.github/workflows/nix-linux-cu10.yml#L32